tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -1241,16 +1241,26 @@ def test_size_1d(f):
     def test_size_2d(f):
         return f.size
 
+    @types('int[:]','int')
+    def test_size_axis(f, axis):
+        return f.size
```

Comment:
This will not test the argument that you are looking at. The class containing the untested code is designed to implement this function: https://numpy.org/devdocs/reference/generated/numpy.ma.size.html#numpy.ma.size

In order to test the code you must implement an example where the `axis` argument is used. You can't just use the `size` function, have a variable called `axis` nearby and hope that the two influence one another

Reply:
Thank you for reviewing my code 🙏.
I'm just testing some hypotheses. About how tests work. And who's calling who.

Reply:
You can also use tests locally to check this with prints. The `-s` option of `pytest` will ensure that the prints aren't swallowed. See https://docs.pytest.org/en/7.1.x/how-to/capture-stdout-stderr.html